Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -359,15 +363,17 @@
   - BANANA HONK
   - BEEPSKY SMASH
   - BLOODY MARYS
+  - BLOOD
   - DOCTOR'S DELIGHT
+  - FOURTEEN-LOKO
   - GARGLE BLASTERS
-  - LEAN
```

Comment:
why?

Reply:
Not in the game anymore, renamed to "grape juice"